Docker

srandall97 opened the pull request:
I added support for Docker and also added some documentation about setting up a database and Quill for production. I kept the documentation as general as possible and tried to mainly rely on links to other sources. I wasn't sure whether to provide command line instructions for Docker but I have tentatively decided to not include them.

srandall97 committed:
add docker support

srandall97 committed:
package-lock.json changed?

srandall97 committed:
Merge branch 'readme-nav' into docker

srandall97 committed:
add documentation for deployment

krubenok commented on Dockerfile:
```diff
@@ -0,0 +1,10 @@
+WORKDIR /usr/app/src
+COPY . .
+EXPOSE 3000
+RUN apk add --no-cache --virtual .gyp python make g++
```
Do we need to this now that the dependency has been updated/removed?

srandall97 replied:
No I guess we don't need it anymore

krubenok reviewed (commented):
Looks good. Need to get back to my computer and run this first. Will check back in on this tomorrow AM.

krubenok committed:
WIP. Add docker compose and new dockerfile

The application launches but it can't connect to the mongoDB started in docker. WIP.

srandall97 committed:
Merge branch 'master' into docker

srandall97 committed:
did final on package-lock before merge

srandall97 commented:
There were some weird diffs with the package-lock.json so I merged master -> docker and ran a check to make sure everything seems to be working. Looks good now so I'll merge.
